src/main/java/org/springframework/data/redis/repository/configuration/EnableRedisRepositories.java

```diff
@@ -52,7 +53,7 @@
 @Documented
 @Inherited
 @Import(RedisRepositoriesRegistrar.class)
-@QueryCreatorType(RedisQueryCreator.class)
+@QueryCreatorType(value = RedisQueryCreator.class, repositoryQueryType = RedisPartTreeQuery.class)
```

Comment:
we should register `RedisPartTreeQuery` for reflective access (`MemberCategory.INVOKE_DECLARED_CONSTRUCTORS`, `MemberCategory.INVOKE_DECLARED_METHODS`) in `RepositoryRuntimeHints`.